ui/page/overview/overview_layout.go

```diff
@@ -246,6 +247,12 @@ func (pg *AppOverviewPage) HandleUserInteractions() {
 		pg.isBackupModalOpened = true
 	}
 
+	autoSync := pg.WL.Wallet.ReadBoolConfigValueForKey(load.AutoSyncConfigKey)
+	pg.autoSync.SetChecked(false)
```

Comment:
`pg.autoSync` should be `pg.autoSyncSwitch` so it is immediately obvious what that property is. `autoSync` as used in line 250 makes sense to represent a boolean variable, not a switch.